### ISSUE TITLE
PyPi ColoredNoise doesn't match GitHub

### ISSUE THREAD

**Issue by wilke0818** [asteroid-team/torch-audiomentations#167]
When attempting to use ColoredNoise with the same values of f_min and f_max, I get the following error:
raise ValueError("Uniform is not defined when low>= high")
ValueError: Uniform is not defined when low>= high

However, in colored_noise.py on the Github, this is mitigated through a conditional statement but when I look at the torch-audiomentations code that I have locally, the if-statement is not there.

Looks like it was merged in #162

**Comment by iver56** [comment]
Yeah, that has not been released yet. If you want, you can install the "bleeding edge" directly from Github instead of PyPi. Or make a fork. It's up to you 😉 Good luck, have fun
